minho/programmers/Level_1/3일차/K번째_수.js

```diff
@@ -0,0 +1,9 @@
+function solution(array, commands) {  
+    let ar = [];
```

Comment:
```suggestion
    const arr = [];
```
배열은 const로 써주는게 좋고 배열 변수명은 arr을 많이씁니다